guildwars2/guild/general.py

```diff
@@ -65,8 +79,7 @@ class GeneralGuild:
     @guild.command(name="members", usage="<guild name>")
     @commands.cooldown(1, 20, BucketType.user)
     async def guild_members(self, ctx, *, guild_name=None):
-        """Get list of the first 20 members and their ranks.
-        Only displays the highest ranks.
+        """Get list of members and their ranks.
```

Comment:
Maybe change it to "Shows a list of members and their ranks" for consistency

Reply:
Changed helper message